tools/component-builder/index.js

```diff
@@ -30,7 +30,7 @@ const build = gulp.series(
 
 const buildLite = gulp.series(
   clean,
-  css.buildVars
+  css.buildIndexVars
```

Comment:
This little change makes restarting the dev server 50 seconds faster.

HOWEVER, it means that undefined variable usage won't be caught until you do a full build!

Let's discuss whether this is a desired change. We could make it run `bakeVars` on change, so at least undefined variable usage will be caught while developing...

Reply:
I make `bakeVars` run when files are changed during development, it also runs when the build is done ahead of release and will block the release if it fails.

This should speed up development and we still should catch undefined vars.